### PR TITLE
Profile and bookmark page

### PR DESCRIPTION
**Pull request by alexWaligorski**
check please!

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -20,6 +20,8 @@ header {
   z-index: 1;
 }
 
+/* Question-Section */
+
```
Comment (liebeckm) [inline css/styles.css:24]: use different CSS files to seperate styles and make the arrangement more clear

File: css/styles.css [hunk 2]
```diff
@@ -110,3 +137,117 @@ footer {
...
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+/* The switch - the box around the slider */
+.dark-mode__switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
```
Comment (liebeckm) [inline css/styles.css:207]: maybe working with rem, % etc. is better due to responsibility issues (applies to the whole file)?

File: bookmark.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```
Comment (liebeckm) [inline bookmark.html:1]: nice done! :)

File: css/styles.css (hunk 1 above)
Comment (alexWaligorski) [reply css/styles.css:24]: will do!

File: css/styles.css (hunk 2 above)
Comment (alexWaligorski) [reply css/styles.css:207]: I will consider this! sometimes it is not nescessary but I will have a close look as soon as I implement my media queries. Thanks!!

File: bookmark.html (hunk 3 above)
Comment (alexWaligorski) [reply bookmark.html:1]: <3

**Pull request merged**